src/native/minipal/time.c

```diff
@@ -56,6 +62,9 @@ inline static void YieldProcessor(void)
 }
 
 #define tccSecondsToNanoSeconds 1000000000      // 10^9
+#define tccSecondsToMillieSeconds 1000          // 10^3
```

Comment:
Is `MillieSecond` a typo that keeps getting copied around (should be `MilliSecond`), or is it actually an established name?

Reply:
It was copied from both pal and mono implementation. It seems that French doesn't spell so either.